Frameworks/CoreText/CTLine.mm

```diff
@@ -322,8 +347,7 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
  @Status Interoperable
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
-    _CTLine* line = static_cast<_CTLine*>(lineRef);
-
+    __CTLine* line = const_cast<__CTLine*>(lineRef);
```

Comment:
Just move this into a CTLine::GetTypographicBounds(...) so it doesn't need to be mutated like this.  Since the values are computed lazily just make them `mutable` so it can stay const.